Apps/APN-UIKit/APN UIKit/AppDelegate.swift

```diff
@@ -85,31 +87,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 }
 
 extension AppDelegate: UNUserNotificationCenterDelegate {
-    // Delegate called when user responds to a notification. Set delegate in
-    // application:didFinishLaunchingWithOptions: method.
     func userNotificationCenter(_ center: UNUserNotificationCenter, didReceive response: UNNotificationResponse, withCompletionHandler completionHandler: @escaping () -> Void) {
-        let handled = MessagingPush.shared.userNotificationCenter(
-            center,
-            didReceive: response,
-            withCompletionHandler: completionHandler
+        CustomerIO.shared.track(
```

Comment:
Is there a reason to add manual push tracking here?

Reply:
This isn't push tracking, but instead simply tracking an event for easier QA testing. 

We expect that when clicking on a push that was not sent by CIO, the host app is able to receive the push notification and handle it. By tracking an event, this makes our QA testing easier so you click on a push and expect to see an event tracked.